MVVM Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -5,6 +5,9 @@
 import UIKit
 
 public final class FeedViewController: UITableViewController, UITableViewDataSourcePrefetching {
+	
+	@IBOutlet weak var errorView: ErrorView!
```

Comment:
This property doesn't need to be `weak`. Also, the setter can be private to prevent clients from changing the instance:

```suggestion
	@IBOutlet private(set) var errorView: ErrorView!
```